node/node.go

```diff
@@ -217,11 +217,50 @@ func NewNode(config *cfg.Config,
 		return nil, fmt.Errorf("Error starting proxy app connections: %v", err)
 	}
 
+	// EventBus and IndexerService must be started before the handshake because
+	// we need index the txs of handshaker replayed block which might not done when last time node stopped
+	// i.e. endblocker panic OR stop node process after save block to db but before index tx
```

Comment:
Suggestion (change to):
// EventBus and IndexerService must be started before the handshake because
// we might need to index the txs of the replayed block as this might not have happened 
// when the node stopped last time (i.e. the node went stopped after it saved the block 
// but before it indexed the txs, or, the endblocker panic'ed)

Reply:
done